File: javascript/evenpal.js
//working...success...
console.log("we will find evenpal no.\n enter a no.");
const prompt=require('prompt-sync')();
var i=parseInt(prompt());
var copy=i,copy2=i;
var rev=0,sum=0;
while(copy!=0){
	sum+=(copy%10);
	copy=parseInt(copy/10);
}
while(copy2!=0){
	rev=(rev*10)+(copy2%10);
	copy2=parseInt(copy2/10);
}
if (rev==i && sum%2==0){
	console.log(`${i} is evenpal no.`);}
else{console.log(`${i} is not a evenpal no.`);
}

File: java/evenpal.java
import java.util.*;
/*working...
success...*/
public class evenpal{
	public static void main(String f[]){
		Scanner nrt=new Scanner(System.in);
		System.out.println("enter a number");
		int n=nrt.nextInt();
		int copy1=n,sumofdigits=0;
		//reverse the number
		while(copy1!=0){
			sumofdigits+=(copy1%10);
			copy1/=10;
		}
		int copy2=n,rev=0;
		while(copy2!=0){
rev= (rev*10)+(copy2%10);
			copy2/=10;
		}
		System.out.println("reverse--- "+rev);
		if(sumofdigits%2==0 && rev==n)System.out.println("evenpal");
		else System.out.println("not evenpal");
	}
}